docs/FAQ.md

```diff
@@ -104,7 +104,9 @@ files you want to keep around to match the pattern.
 
 If `$EDITOR` integration is important, something like `scratchpad.*` may be more
 helpful, as you can keep the filename extension intact (it
-matches `scratchpad.md`, `scratchpad.rs` and more).
+matches `scratchpad.md`, `scratchpad.rs` and more). Another option is to add a 
+directory to the global `.gitignore` which then stores all your temporary files
+and notes. To achieve this add `<your-directory>/` to the`.git/info/exclude`.
```

Comment:
If you prefer the 'the', there's also a space needed here, so you could choose this version too.

```suggestion
and notes. To achieve this add `<your-directory>/` to the `.git/info/exclude`.
```